### ISSUE TITLE
Task ` chp.insert-chp-egon100re` fails [CI | SH]

### ISSUE THREAD

**Issue by nesnoj** [openego/eGon-data#912]
```
[2022-09-06 10:27:49,217] {taskinstance.py:1150} ERROR - (psycopg2.errors.UndefinedTable) relation "supply.egon_chp_plants" does not exist
LINE 2:         DELETE FROM supply.egon_chp_plants
```

**Comment by ClaraBuettner** [comment]
Should be fixed by 72c0e18efab5b409cd53620488e03c9c51393952

**Comment by nesnoj** [comment]
Yepp, works!
